packages/api/src/routes/medical/internal-facility.ts

```diff
@@ -59,27 +63,20 @@ router.put(
       cwApproved: facilityDetails.cwApproved,
     };
     let facility: Facility;
-    let facilityCurrentActive = false;
     if (facilityDetails.id) {
-      const currentFacility = await getFacilityOrFail({ cxId, id: facilityDetails.id });
-      facilityCurrentActive = currentFacility.cqActive;
+      await getFacilityOrFail({ cxId, id: facilityDetails.id });
```

Comment:
Keep loading the facility even though its not needed for the update for Authorization purposes. Should we add a comment?